segmentation/include/pcl/segmentation/impl/unary_classifier.hpp

```diff
@@ -52,14 +52,9 @@
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointT>
-pcl::UnaryClassifier<PointT>::UnaryClassifier () :
-  input_cloud_ (new pcl::PointCloud<PointT>),
-  label_field_ (false),
-  normal_radius_search_ (0.01f),
-  fpfh_radius_search_ (0.05f),
-  feature_threshold_ (5.0)
-{
-}
+pcl::UnaryClassifier<PointT>::UnaryClassifier()
+: input_cloud_(new pcl::PointCloud<PointT>)
```

Comment:
I don't see any reason to construct this empty cloud?

Reply:
Don't you agree on this one or was it just overlooked?

Reply:
Overlooked. (I really don't like the behavior of the review windows now; they close too easily, and it's easy to forget the comments that you haven't finished addressing.)